.github/workflows/cherry-picking.yaml

```diff
@@ -0,0 +1,38 @@
+name: Cherry picking
+on:
+  push:
+    branches:
+      - master
+jobs:
+  cherry_pick:
+    runs-on: ubuntu-latest
+    env:
+      #Variable for the name of the branch to cherry-pick into.
+      #It will remain 'invalid' if no branch is specified
+      destination_branch: 'invalid'
+    steps:
+    - name: checkout
+      uses: actions/checkout@v2
+    #Removes posible conflicting characters on the commit message
+    - name: Remove conflicting chars
+      uses: frabert/replace-string-action@v1.2
+      id: remove_quotations
+      with:
+        pattern: "\""
+        string: ${{github.event.commits[0].message}}
+        replace-with: "-"    
+    #Checks the message looking for a cherry-pick request and extracts the target branch name
+    - name: Check Information
+      id: check-info
+      run: |
+        echo "destination_branch=$(pwsh .\\.github\\scripts\\cherry_pick_check.ps1 "${{ steps.remove_quotations.outputs.replaced }}" )" >> $GITHUB_ENV
+    - name: Create PR to branch
+      #If a target branch was found will run the action
+      if: env.destination_branch != 'invalid'
+      uses: gorillio/github-action-cherry-pick@master
```

Comment:
The only concern for me is if this action will be maintained on the long term, because is not supported by someone like GitHub or Microsoft, on that case I will prefer to create our own script that will do the logic.

What do you think @QilongTang ?

Reply:
@alfredo-pozo You have a good point. I am less worried about this given Git cli commands are very backward compatible. I am hesitate to invest too much in writing our own action not because it's not useful but because of the effort to maintain them. We will rely on a lot of actions to finish our automations so I say we leave it as it is for now and focus on other Dynamo improvements. Thank you for the comment though